src/containers/Reports/Reports.js

```diff
@@ -52,41 +54,55 @@ class Reports extends Component {
         const formatted = Object.keys(data['callersByMonth']).map((el)=>{
             const numCallers = data['callersByMonth'][el]
             const numCalls = data['callsByMonth'][el] || 0
-            return {date: el, Callers: numCallers, Calls: numCalls};
-        });
+            const completed = (numCalls/numCallers * 100).toFixed(1) + "%"
+            return {date: el, Callers: numCallers, Calls: numCalls, Completed: completed};
+        });  
         return formatted.slice(0, 11);
     }
 
+    
     render() {
         const statistics = this.state.statistics;
         const antIconBig = <Icon type="loading" style={{ fontSize: 28 }} spin />
         const antIconHuge = <Icon type="loading" style={{ fontSize: 72 }} spin />
         const districtTitle = this.props.district ? `${displayName(this.props.district)} ` : "";
 
+        let completionRate = 0; 
+        if(statistics && statistics.totalCalls > 0 && statistics.totalCallers > 0)
+                completionRate = (statistics.totalCalls/statistics.totalCallers * 100).toFixed(1); 
```

Comment:
On testing, I'm realizing this calculation is incorrect. The denominator should be the number of reminders sent, not the total number of callers. The numerator should be the number of active callers. For example, if there is one caller who calls her Senators and Rep in January and just her Rep in February, that's 4 calls over one caller, resulting in a completion rate of 400%, when it's actually just 100%.

Here is an example stats response with real data:

```
{
    "totalCalls": 21,
    "totalCallers": 27,
    "recentDayCount": 30,
    "totalRecentCalls": 5,
    "callsByMonth": {
        "2020-02": 2,
        "2020-03": 3,
        "2020-04": 2,
        "2020-05": 5,
        "2020-06": 6,
        "2020-07": 3
    },
    "callersByMonth": {
        "2019-07": 1,
        "2019-08": 1,
        "2019-09": 1,
        "2019-10": 2,
        "2019-11": 2,
        "2019-12": 2,
        "2020-01": 4,
        "2020-02": 20,
        "2020-03": 23,
        "2020-04": 24,
        "2020-05": 24,
        "2020-06": 24,
        "2020-07": 27
    },
    "activeCallersByMonth": {
        "2019-07": 0,
        "2019-08": 0,
        "2019-09": 0,
        "2019-10": 0,
        "2019-11": 0,
        "2019-12": 0,
        "2020-01": 0,
        "2020-02": 2,
        "2020-03": 3,
        "2020-04": 2,
        "2020-05": 5,
        "2020-06": 6,
        "2020-07": 3
    },
    "remindersByMonth": {
        "2019-08": 1,
        "2019-09": 1,
        "2019-10": 2,
        "2019-11": 2,
        "2019-12": 2,
        "2020-01": 4,
        "2020-02": 14,
        "2020-03": 22,
        "2020-04": 24,
        "2020-05": 24,
        "2020-06": 24,
        "2020-07": 24
    },
    "districtId": 593
}
```

Updating the calculation here should be quite easy based on the way the data is presented in that response.

Note that this comment applies to the similar calculation a few lines above.